docs/conf.py

```diff
@@ -1,11 +1,14 @@
 # -*- coding: utf-8 -*-
 # icalendar documentation build configuration file
-import pkg_resources
 import datetime
 import os
+import sys
 
+base_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
 on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
+sys.path.append(os.path.join(base_dir, 'src'))
```

Comment:
Is this needed for a successful RTD build?

Reply:
No, it isn't strictly. I added this because I removed the `.` line from your requirements file. I believe editing the sys path is the preferred way of adding the module to the path. Depending on your setup, you may not need this.